app/src/main/java/com/arctouch/codechallenge/ui/home/HomeViewModel.kt

```diff
@@ -0,0 +1,62 @@
+package com.arctouch.codechallenge.ui.home
+
+import android.annotation.SuppressLint
+import android.arch.lifecycle.ViewModel
+import android.util.Log
+import com.arctouch.codechallenge.api.TmdbApi
+import com.arctouch.codechallenge.data.Cache
+import com.arctouch.codechallenge.model.Movie
+import com.arctouch.codechallenge.util.Logger
+import io.reactivex.Observable
+
+class HomeViewModel : ViewModel() {
+    private var TAG = Logger.tag
+
+    var cacheMovies = mutableListOf<Movie>()
```

Comment:
What's the difference between `var cacheMovies = listOf<Movie>()` and `val cacheMovies = mutableListOf<Movie>()`? Why did you use `var` +  `mutableListOf()`? Why does that matter?

Reply:
**What's the difference between [var cacheMovies = listOf<Movie>()](url) and `val cacheMovies = mutableListOf<Movie>()`?**  According to the documentation the List is read-only and mutableList gives us full control access, inserting, removing, editing. 
Why did you use `var` + `mutableListOf()`? Because I will change the data and the `val` do not allow this. Why does that matter?This is because Kotlin uses these names (`val` and `var`) to define variables of the final type, and also the access question via `GET` and `SET`, which in the case of Kotlin are authomatically defined methods